src/yuzu/configuration/configure_input_player_widget.cpp

```diff
@@ -220,13 +220,34 @@ void PlayerControlPreview::UpdateInput() {
 
     if (input_changed) {
         update();
+        ControllerInput input{
+            .axis_values =
+                {std::pair<float, float>{axis_values[Settings::NativeAnalog::LStick].value.x(),
+                                         axis_values[Settings::NativeAnalog::LStick].value.y()},
+                 std::pair<float, float>{axis_values[Settings::NativeAnalog::RStick].value.x(),
+                                         axis_values[Settings::NativeAnalog::RStick].value.y()}},
+            .button_values = button_values,
+            .changed = true,
+        };
+
+        if (controller_callback.input != nullptr) {
+            controller_callback.input(std::move(input));
+        }
+    }
+
+    if (controller_callback.update != nullptr) {
+        controller_callback.update(std::move(true));
```

Comment:
```suggestion
        controller_callback.update(true);
```

The move does nothing here.